scripts/script.js

```diff
@@ -53,6 +54,12 @@ cocktailApp.prepareSubmitListener = function () {
 };
 
 cocktailApp.getRandomCocktail = function() {
+  // adding pop-up function
+  cocktailApp.popup.classList.add('popup__container--open')
```

Comment:
The logic you've got this wrapped in is super clean and elegant. Nicely done!

Can we put this logic in its own function that _calls_ .getRandomCocktail() to make sure that each function does just one thing?